cmd/localCI/config/configuration.toml

```diff
@@ -0,0 +1,7 @@
+[[github.com]]
+owner = "clearcontainers"
+repos = ["tests", "runtime", "proxy"]
```

Comment:
Can we have a nicer way to list the repos? If you want to use TOML, that's sounds fine to me, but let's make Repositories proper objects so we can add per-repository config options int the future.

You are using an "array of table" which is really an array of objects. You're objects are a bit off here, what about:

```
[[Repositories]]
name = runtime
url = https://github.com/...
test_action = make check
...

[[Repositories]]
name = proxy
url = https://github.com/...

...

[[Repositories]]
name = ...
url = ...
```